ui/components/src/ui-components/propertygrid/component/PropertyGridCommons.ts

```diff
@@ -62,8 +62,10 @@ export interface CommonPropertyGridProps extends CommonProps {
   /** Callback for when properties are updated @beta */
   onPropertyUpdated?: (args: PropertyUpdatedArgs, category: PropertyCategory) => Promise<boolean>;
 
-  /** Callback for when links in properties are being clicked @beta */
-  onPropertyLinkClick?: (property: PropertyRecord, text: string) => void;
+  /** Callback for when links in properties are being clicked
+   * @deprecated Should override data provider and set it on [[PropertyRecord]] instead
+   */
+  onPropertyLinkClick?: (text: string) => void;
```

Comment:
I believe we don't have to change declaration of this function, do we? Also, the `@beta` tag should stay.